content/en/blog/2023/sig-testing-spotlight.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: blog
+title: "Spotlight on SIG Testing"
+slug: sig-testing-spotlight-2023
+date: 2023-11-24
+author: "Sandipan Panda"
+---
+
+Welcome to another edition of the SIG Spotlight blog series, where we
```

Comment:
```suggestion
Welcome to another edition of the _SIG spotlight_ blog series, where we
```

This'll avoid readers thinking that we have a Spotlight special interest group, even momentarily.